requirements.txt

```diff
@@ -102,5 +100,6 @@ apache-airflow-providers-imap==1.0.1
 apache-airflow-providers-sqlite==1.0.2
 apache-airflow-providers-celery==1.0.1
 redis~=3.2
+hdfs~=2.6.0
```

Comment:
Why put `hdfs` in requirements while `flink` in requirements-dev?

Reply:
The S3 dependencies are in requirements, therefore the hdfs could be same as the S3. And the Flink dependencies depend on the environment of the users and the unit tests need the Flink dependencies to run the tests for the Flink Job plugin.